README.md

```diff
@@ -65,6 +65,8 @@ Given a path specified by `from`, upload it to the Swift container. The path mus
 
 * `from`: *Required.* A regexp specifying the file to upload.
 
+* `SegmentContainer` : *Required* Name of the container where segments should be stored. Segments will be created for files larger than 1 GB.
```

Comment:
Is this really required? That would mean the change is not backward compatible, which would make me sad.

Reply:
For uploading segments we need a segment container. A second option would be that default is to append _segments to the default container (like in swift cli client ) . SegmentContainer would be optional in this case.

Reply:
My point is that if I only upload small files i never need to worry about a segment container. So the `SegmentContainer` parameter is not really required. It is only required if you have a large uploads or specifically want to segment your uploads.